package.json

```diff
@@ -2,23 +2,68 @@
   "name": "@api-platform/api-doc-parser",
   "version": "0.16.8",
   "description": "Transform an API documentation (Hydra, OpenAPI, GraphQL) in an intermediate representation that can be used for various tasks such as creating smart API clients, scaffolding code or building administration interfaces.",
-  "files": [
-    "*.md",
-    "lib",
-    "src"
+  "keywords": [
+    "api",
+    "api-platform",
+    "documentation",
+    "hydra",
+    "openapi",
+    "graphql",
+    "jsonld",
+    "json-schema",
+    "typescript",
+    "client",
+    "codegen",
+    "scaffolding"
```

Comment:
It's a parser, not a code generator, isn't it?

Reply:
Not its not, but it can be used to build one, like said in the readme:

> This data structure can then be used for various tasks such as creating smart API clients, scaffolding code or building administration interfaces.

Reply:
Yes, but since it's not a code generator, these keywords shouldn't be present IMO.

Reply:
No problem, I'll remove them